fastly.toml

```diff
@@ -10,8 +10,8 @@ language = "rust"
 [setup]
   [[setup.backends]]
     name = "backend_name"
-    prompt = "Backend able to serve `/solutions/` path"
-    address = "developer.fastly.com"
+    prompt = "Backend able to serve `/fastly` path"
+    address = "github.com"
```

Comment:
Do we really want to be configuring all default Compute@Edge services with a backend pointing at github.com? 🤔

Reply:
If I'm understanding the concern correctly, wouldn't this same point have applied for `httpbin.org`?

Is it preferable to avoid having a starter kit that can be run without the user having to do anything, if we're concerned on the side-effect this will have on whatever backend we determine should be the default.

Reply:
> wouldn't this same point have applied for httpbin.org?

Yes, but httpbin.org is a service kind of designed to receive spammy requests from the far corners of the internet — I'm not sure the same can be said of github.com.

> Is it preferable to avoid having a starter kit that can be run without the user having to do anything, if we're concerned on the side-effect this will have on whatever backend we determine should be the default.

I'm all for starter kits that work out of the box 👍  It's just that if we're going to predefine backends, and we want to be good Netizens or whatever, they should probably point to stuff that we own and maintain. IMO.

Reply:
> they should probably point to stuff that we own and maintain

Yeah that ended up being tricky. sites like developer.fastly.com and docs.fastly.com both had issues being proxied to because of (just as an example) custom fonts failing to load. I tried the apex fastly.com but that then redirects to the fastly.com rather than proxy the request.

So if we're to keep with the example having two backends I'll need either something we own that we can actually proxy to or use _another_ common service like httpbin (not sure what though).

I could try proxying to `fiddle.fastlydemo.net` actually (only just thought of that one).

Reply:
Scratch fiddle, that doesn't work when proxied to 😬

Reply:
@peterbourgon I just pushed a commit to switched the backend from `github.com` to `reqbin.com`. Not as well known as `httpbin.org` but it's a similar service in the sense of it's designed to receive spammy requests. Opinions?

Reply:
Why don't those other Fastly properties work? Do you change the Host header?

Reply:
Nope I don't change the header. I think it's to do with the various JS scripts that need to be downloaded to render the page correctly (i.e. the `main.rs` doesn't set up any handlers for those URLs and so the requests 404)...

<img width="1920" alt="Screenshot 2021-09-07 at 09 28 24" src="https://user-images.githubusercontent.com/180050/132311726-4514e43d-a2f9-4ce7-9d42-113821fd77c5.png">

Reply:
Just to confirm that @fastly/developer-relations is happy with using `reqbin.com` for the purposes of this Starter Kit. So I'll look to merge this PR today.